src/storage/mod.rs

```diff
@@ -999,6 +999,15 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
             )
         });
         let priority_tag = get_priority_tag(priority);
+        keys.sort();
+        let mut i = 1;
```

Comment:
Why not use the `dedup` interface of `Vec`? The current implementation is O(n^2) time complexity.